src/components/contactPicker/ContactPicker.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export const ContactPicker = ({
+  name,
+  onChange,
+  contacts}) => {
```

Comment:
don't forget to run your code through prettier before uploading to Author

Reply:
Done